core/src/doc/document.rs

```diff
@@ -133,6 +133,7 @@ impl<'a> Document<'a> {
 				// We can create the table automatically
 				run.add_and_cache_ns(opt.ns(), opt.strict).await?;
 				run.add_and_cache_db(opt.ns(), opt.db(), opt.strict).await?;
+				// TODO: should there be TableType as param here?
```

Comment:
Can you explain this comment @RaphaelDarley ?

Reply:
this is a vestigial comment, in a previous implementation I provided an additional argument to add_and_cache_tb which was a table type and would be used to specify the type of an implicitly defined table, but now these are always Any so its unnecessary.